problems/objects.js

```diff
@@ -226,16 +369,24 @@ const updateList = () => {};
  * @returns {Object} The Object without the key.
  */
 
- const deleteKey = () => {};
-
+const deleteKey = (obj, key) => {
+  delete obj[key]
+  return obj
+};
 
- /**
-  * Takes in an object and returns the number of
-  * properties it has.
-  * @param {Object} obj
-  * @returns {number} Number of properties.
-  */
- const propertyCount = () => {};
+/**
+ * Takes in an object and returns the number of
+ * properties it has.
+ * @param {Object} obj
+ * @returns {number} Number of properties.
+ */
+const propertyCount = (obj) => {
+  let propertyCounter = 0
+  for (let i = 0; i < Object.keys(obj).length; i++) {
+    propertyCounter += 1
+  }
+  return propertyCounter
```

Comment:
I see you have this code in your for loop: Object.keys(obj).length 
what does it do and how could it help us solve this problem better?